homeassistant/components/icloud/translations/en.json

```diff
@@ -29,7 +29,8 @@
                 "data": {
                     "password": "Password",
                     "username": "Email",
-                    "with_family": "With family"
+                    "with_family": "With family",
+                    "gps_accuracy_threshold": "Discard inaccurate location updates above threshold (meters)"
                 },
```

Comment:
@ntilley905 Should this be removed as well? Will localise use the `strings.json` or this file?

Reply:
Adding the text to `en.json` is fine, there just shouldn't be translations done via direct editing.